wordpress-mobile/WordPress-Android
# Issue/7533 add tab analytics

### Fix
Add analytics tracking for swiping between tabs as described in https://github.com/wordpress-mobile/WordPress-Android/issues/7533.  The [`switch` statement](https://github.com/wordpress-mobile/WordPress-Android/pull/7535/files#diff-1cf6428dcfe0d429dfdcf7398ef4c056R223) was included in order to easily add analytics for each tab specifically if needed in the future.

### Test
0. Set breakpoint [here](https://github.com/wordpress-mobile/WordPress-Android/compare/issue/7533-add-tab-analytics?expand=1#diff-1cf6428dcfe0d429dfdcf7398ef4c056R229).
1. Launch app.
2. Swipe between tabs.
3. Notice breakpoint is hit.
4. Tap tab.
5. Notice breakpoint is not hit.

#### Note
This is not meant to be a permanent change.  This is to analyze the usage of swiping between tabs.  Once the analysis is completed, these changes should be reverted.
